telecine/src/main/res/values/strings.xml

```diff
@@ -8,9 +8,9 @@
   <string name="tile_name">Launch Telecine</string>
   <string name="clear">Clear</string>
   <string name="record">Record</string>
-  <string name="countdown_one">1…</string>
-  <string name="countdown_two">2…</string>
-  <string name="countdown_three">3…</string>
+  <string name="countdown_one" translatable="false">1…</string>
+  <string name="countdown_two" translatable="false">2…</string>
+  <string name="countdown_three" translatable="false">3…</string>
```

Comment:
Yes I thought about that but I don't think they are that often used in the IT world these days.
Also, there was no such translations in the resources.
If we really want tu use arabic/hindu numerals, we should use the built-in Locale mechanism (with `String.format()` with current `Locale`)